contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -206,33 +215,99 @@ contract Dashboard is AccessControlEnumerable {
         if (msg.value == 0) revert ZeroArgument("msg.value");
     }
 
+    /**
+     * @notice Grants multiple roles to a single account.
+     * @param _account The address to which the roles will be granted.
+     * @param _roles An array of bytes32 role identifiers to be granted.
+     * @dev Performs the role admin checks internally.
+     */
+    function grantRoles(address _account, bytes32[] memory _roles) external {
+        if (_account == address(0)) revert ZeroArgument("_account");
+        if (_roles.length == 0) revert ZeroArgument("_roles");
+
+        for (uint256 i = 0; i < _roles.length; i++) {
+            grantRole(_roles[i], _account);
+        }
+    }
+
+    /**
+     * @notice Batch-grants a single role to a single account.
+     * @param _accounts An array of addresses to which the roles will be granted.
+     * @param _roles An array of bytes32 role identifiers to be granted.
+     */
+    function grantRoles(address[] memory _accounts, bytes32[] memory _roles) external {
+        if (_accounts.length == 0) revert ZeroArgument("_accounts");
+        if (_roles.length == 0) revert ZeroArgument("_roles");
+        if (_accounts.length != _roles.length) revert UnequalLengths();
+
+        for (uint256 i = 0; i < _accounts.length; i++) {
+            grantRole(_roles[i], _accounts[i]);
+        }
+    }
+
+    /**
+     * @notice Revokes multiple roles from a single account.
+     * @param _account The address from which the roles will be revoked.
+     * @param _roles An array of bytes32 role identifiers to be revoked.
+     */
+    function revokeRoles(address _account, bytes32[] memory _roles) external {
+        if (_account == address(0)) revert ZeroArgument("_account");
+        if (_roles.length == 0) revert ZeroArgument("_roles");
+
+        for (uint256 i = 0; i < _roles.length; i++) {
+            revokeRole(_roles[i], _account);
+        }
+    }
+
+    /**
+     * @notice Batch-revokes a single role from a single account.
+     * @param _accounts An array of addresses from which the roles will be revoked.
+     * @param _roles An array of bytes32 role identifiers to be revoked.
+     */
+    function revokeRoles(address[] memory _accounts, bytes32[] memory _roles) external {
+        if (_accounts.length == 0) revert ZeroArgument("_accounts");
+        if (_roles.length == 0) revert ZeroArgument("_roles");
+        if (_accounts.length != _roles.length) revert UnequalLengths();
+
+        for (uint256 i = 0; i < _accounts.length; i++) {
+            revokeRole(_roles[i], _accounts[i]);
+        }
+    }
+
     /**
      * @notice Transfers ownership of the staking vault to a new owner.
      * @param _newOwner Address of the new owner.
      */
-    function transferStVaultOwnership(address _newOwner) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
-        _transferStVaultOwnership(_newOwner);
+    function transferOwnership(address _newOwner) external virtual {
```

Comment:
StVault was there for a reason. Without it, you may think that Dashboard has some ownership, which can be transferred.

Reply:
I thought we rejected the "StVault" term